src/transaction/log_page_buffer.c

```diff
@@ -4736,22 +3324,22 @@ logpb_flush_all_append_pages (THREAD_ENTRY * thread_p)
     }
 
   /* now flush the nxio_lsa page... unless it is the page of header for incomplete log record */
-  if (log_Pb.partial_append.status == LOGPB_APPENDREC_SUCCESS
-      || (log_Gl.append.nxio_lsa.pageid != log_Gl.append.prev_lsa.pageid))
+  nxio_pageid = log_Gl.append.get_nxio_lsa ().pageid;
+  if (log_Pb.partial_append.status == LOGPB_APPENDREC_SUCCESS || (nxio_pageid != log_Gl.append.prev_lsa.pageid))
     {
       assert (log_Pb.partial_append.status == LOGPB_APPENDREC_SUCCESS
 	      || log_Pb.partial_append.status == LOGPB_APPENDREC_PARTIAL_FLUSHED_END_OF_LOG);
 
-      bufptr = &log_Pb.buffers[logpb_get_log_buffer_index (log_Gl.append.nxio_lsa.pageid)];
+      bufptr = &log_Pb.buffers[logpb_get_log_buffer_index (log_Gl.append.get_nxio_lsa ().pageid)];
```

Comment:
I guess it should be nxio_pageid not log_Gl.append.get_nxio_lsa

Reply:
fixed